scripts/config/cspell/dictionaries/qiskit.txt

```diff
@@ -249,6 +249,7 @@ nelder
 nonlocality
 nontrivially
 nqubits
+nshots
```

Comment:
This should probably not go in the global dictionary?

Reply:
hilariously, it was there twice. I took it out and moved it to the file